src/utils/log.rs

```diff
@@ -58,3 +61,56 @@ unsafe extern "C" fn log_callback(importance: wlr_log_importance,
     });
     RUST_LOGGING_FN(importance, message);
 }
+
+pub struct Logger;
+
+impl Logger {
+    pub fn init(level: LevelFilter) {
```

Comment:
Document this function, include a section like so:

```
/// # Panicking
/// This function should only be called once in the lifetime of the program. 
/// If it's called multiple times it will panic.
```